templates/CRM/CiviAwards/Form/AwardReview.tpl

```diff
@@ -39,11 +39,7 @@
         <div class="form-group {$form_group_class}">
           {if $isReviewFromSsp}
             <h1 class="header-pager">
-              {if $isViewAction}
-                {$sourceContactName}
-              {else}
-                {$form.source_contact_id.html}
-              {/if}
+              {$caseContactDisplayName}
```

Comment:
We don't need the else part of the if statement above?

Reply:
Don't think so. We need to show the same contact display name to view and non-view pages. Right?

Reply:
@tunbola

Reply:
The logic above shows that on Non view pages, i.e on create, A form field should show. Can you confirm create and view works fine after this change?

Reply:
Yeas. Both the view page and form page works fine. 
<img width="1235" alt="Screenshot 2020-07-07 at 9 23 09 PM" src="https://user-images.githubusercontent.com/3340537/86808130-38479a00-c098-11ea-9e77-91ca06fed82a.png">
<img width="1287" alt="Screenshot 2020-07-07 at 9 23 52 PM" src="https://user-images.githubusercontent.com/3340537/86808155-3da4e480-c098-11ea-9b8d-ed69bd7c0d0e.png">